docs/included.md

```diff
@@ -280,7 +344,15 @@
   - 
     [Hanssen and Kuipers discriminant (WWRP/WGNE Joint Working Group on Forecast Verification Research)](https://www.cawcr.gov.au/projects/verification/#HK)
 * -  
-    - Probability of Detection (POD) (Hit Rate, True Positive Rate, Sensitivity)
+    - Precision (Success Ratio)
+  -
+    [API](https://scores.readthedocs.io/en/latest/api.html#scores.categorical.BasicContingencyManager.precision)
+  - 
+    [Tutorial](https://scores.readthedocs.io/en/latest/tutorials/Binary_Contingency_Scores.html)
+  - 
+    [Wikipedia](https://en.wikipedia.org/wiki/Precision_and_recall)
```

Comment:
Not sure that this matters. Success Ratio has a different reference ```[Success ratio (WWRP/WGNE Joint Working Group on Forecast Verification Research)](https://www.cawcr.gov.au/projects/verification/#SR)```

Should they:
- Have the same reference,
- Both entries list both references, or
- Have different references?

Reply:
Yes I was aware that they have different references. I was unsure what to do. This was my thinking:

I think that if someone is looking to use "precision" they are probably interested in using it in the context of "precision recall" and may want to build "precision recall diagrams". On the other hand, if someone is looking for "success ratio" the cawcr reference is more appropriate.

Reply:
I see. Would it be worthwhile listing both references in both locations - then people can pay attention to whichever (or both) as they see fit? But I am also fine with how it is now.

Reply:
I added both references (but only under `precision` and `recall` - not their other names)

Reply:
Sounds good.